test/cluster/test_repair.py

```
@@ -854,3 +854,39 @@ async def test_tablet_migration_barrier_does_not_abort_vnode_repair(manager: Man
+                         ("1x", "cannot parse integer")]:
+        params = {"columnFamilies": "tbl", "ranges_parallelism": value}
+        with pytest.raises(HTTPError, match=error):
+            await manager.api.client.post_json(f"/storage_service/repair_async/ks",
+                                               host=servers[0].ip_addr, params=params)
+
+    # A positive value, and omitting the option altogether, must keep working.
+    for params in [{"columnFamilies": "tbl", "ranges_parallelism": "1"},
+                   {"columnFamilies": "tbl"}]:
+        sequence_number = await manager.api.client.post_json(f"/storage_service/repair_async/ks",
+                                                             host=servers[0].ip_addr, params=params)
+        status = await manager.api.client.get_json(f"/storage_service/repair_status",
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)